config/base/kustomization.yaml

```diff
@@ -24,5 +24,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: cloudfoundry/cf-k8s-api
-  newName: cloudfoundry/cf-k8s-api
-  newTag: latest
+  newName: cf-k8s-api
+  newTag: bc82f627-9e1f-4560-9074-fd3571222450
```

Comment:
Is this something we'll want to keep on main, or should we change it back to "latest" once it's merged?

Reply:
This happens as a result of providing the `IMG` environment variable when using the Make targets. Using `deploy` as an example, we see `cd config && $(KUSTOMIZE) edit set image cloudfoundry/cf-k8s-api=${IMG}` which directly edits the `kustomization.yaml`. This could be unset and re-kustomized to remove this change.

Only a longer time scale, we should probably have a make target that goes in an sets it back to default and generates our reference yaml.

Reply:
I shouldn't have committed this. I'll get it reset.

Also agree with Birdrock that it would be good to have something that resets it automatically.